dotCMS/src/main/webapp/html/js/dotcms/dijit/form/FileAjaxUploader.js

```diff
@@ -140,284 +148,264 @@ dojo.declare("dotcms.dijit.form.FileAjaxUploader", [dijit._Widget, dijit._Templa
-        dojo.style(this.fileUploadRemoveButton, { display: 'none' });
-        dojo.style(this.fileUploadInfoButton, { display: 'none' });
-        if(document.getElementById('fileTextEditorDiv')){
-            dojo.style("fileTextEditorDiv", { display: 'none' });
+        this.fileNameDisplayField.innerHTML = this.fileInputField.value = "";
+        this.fileNameField.value = "---removed---";
+
+        dojo.style(this.fileNameDisplayField, { display: "none" });
+        dojo.style(this.fileUploadForm, { display: "" });
+        dojo.style(this.fileUploadRemoveButton, { display: "none" });
+        dojo.style(this.fileUploadInfoButton, { display: "none" });
+        if (document.getElementById("fileTextEditorDiv")) {
+            dojo.style("fileTextEditorDiv", { display: "none" });
         }
 
-		this.onRemove(this);
-
+        this.onRemove(this);
     },
 
     _info: function () {
-
+        const host = location.protocol + "//" + location.host;
         var fileInfo = {};
-        fileInfo['fileName'] = this.fileName;
-        fileInfo['path'] = location.protocol +"//"+ location.host + '/dA/';
-
-        fileInfo['path'] += (this.identifier != '0') ? this.idShorty : this.inodeShorty;
-        fileInfo['path'] += (this.id != 'fileAsset') ? '/' +  this.id : "";
-        fileInfo['path'] += "/" + this.fileName +'?language_id='+this.lang;
-
-
-        var html = dojo.replace(this._fileInfoTemplate(), fileInfo);
+        fileInfo["fileName"] = this.fileName;
+        fileInfo["path"] =  host + "/dA/";
+
+        fileInfo["path"] += this.identifier != "0" ? this.idShorty : this.inodeShorty;
+        fileInfo["path"] += this.id != "fileAsset" ? "/" + this.id : "";
+        fileInfo["path"] += "/" + this.fileName + "?language_id=" + this.lang;
+        fileInfo["versionPath"] = host + this.versionPath;
+        fileInfo["idPath"] = host + this.idPath;
+        fileInfo["versionPathLabel"] = this.versionPathLabel
+        fileInfo["idPathLabel"] = this.idPathLabel
+        fileInfo["resourceLink"] = this.resourceLink
+        fileInfo["resourceLinkLabel"] = this.resourceLinkLabel
+
+        var html = dojo.replace(this._fileInfoTemplate({
+            resourceLink: this.resourceLink,
+            idPath: this.idPath,
+            versionPath: this.versionPath
+        }), fileInfo);
 
         this.fileInfoDialog.title = this.fileName;
         var domObj = dojo._toDom(html);
         this.fileInfoDialog.setContent(domObj);
         this.fileInfoDialog.show();
     },
 
-
-    _isFileAsset : function(){
-
-        return (
-            dijit.byId("fileName")
-            &&  dijit.byId("title")
-            &&  dijit.byId("sortOrder")
-            &&  dojo.byId("metaData_field")
-        );
-
+    _isFileAsset: function () {
+        return dijit.byId("fileName") && dijit.byId("title") && dijit.byId("sortOrder") && dojo.byId("metaData_field");
     },
 
+    _postUploadCallback: function (fileData) {
+        var fieldRelatedData = {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid trailing commas in object or array literals](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5396940)